interface/usergroup/user_admin.php

```diff
@@ -96,6 +96,12 @@
           sqlStatement("UPDATE users, facility SET users.facility = facility.name WHERE facility.id = '$tqvar' AND users.id = {$_GET["id"]}");
           //END (CHEMED)
   }
+
+  if ($_GET["role_name"]) {
+    $tqvar = formData('role_name', 'G');
+    sqlStatement("update users set fullscreen_role ='$tqvar' where id = {$_GET["id"]}");
```

Comment:
Sql binding would be better.

Reply:
$query = "UPDATE `users` SET `fullscreen_role` = '".$tqver."' WHERE `id` =...... right?